openssl/openssl.php

```diff
@@ -721,6 +890,26 @@ function openssl_verify($data, $signature, $public_key, $algorithm = OPENSSL_ALG
  */
 function openssl_seal($data, &$sealed_data, array &$encrypted_keys, array $public_key, $cipher_algo = null, &$iv = '') { }
 
+#[PhpStormStubsElementAvailable('8.0')]
+/**
+ * Seal (encrypt) data
+ * @link https://php.net/manual/en/function.openssl-seal.php
+ * @param string $data
+ * @param string &$sealed_data
+ * @param array &$encrypted_keys
+ * @param array $public_key
+ * @param string $cipher_algo
+ * @param string &$iv [optional]
+ * @return int|false the length of the sealed data on success, or false on error.
+ * If successful the sealed data is returned in
+ * <i>sealed_data</i>, and the envelope keys in
+ * <i>env_keys</i>.
+ */
+function openssl_seal($data, &$sealed_data, array &$encrypted_keys, array $public_key, $cipher_algo, &$iv = '')
+{
+}
+
+#[PhpStormStubsElementAvailable('5.3','7.4')]
```

Comment:
Generally such samples can be replaced with `#[PhpStormStubsElementAvailable(to: '7.4')]`